cmake/modules/SwiftSource.cmake

```diff
@@ -200,6 +248,23 @@ function(_compile_swift_files
     endif()
   endforeach()
 
+  set(output_option)
+  if (SWIFTFILE_USE_THREADED_WMO)
+    get_filename_component(output_dir "${first_output}" DIRECTORY)
+    set(output_file_map "${output_dir}/output.json")
+    _generate_output_file_map("${output_file_map}"
+      SOURCES ${source_files}
+      OUTPUTS ${SWIFTFILE_OUTPUT})
+    set(output_option
+      "-output-file-map" "${output_file_map}"
+      "-num-threads" "2")
```

Comment:
Why is 2 threads hard coded here? Needs at least an explanation.

Reply:
:-) This was me testing correctness. I haven't really thought about what the actual right thing is to do here, because I don't want to overwhelm the system with threads if you build all the stdlibs at once. You're definitely right that any choice here needs to be explained, even if it's 1 or NUM_CORES.